policy/datasets.yaml

```diff
@@ -1119,4 +1119,27 @@ deepCoadd_sg_features_tract:
     storage: ParquetStorage
     python: lsst.qa.explorer.parquetTable.ParquetTable
     template: deepCoadd-results/merged/%(tract)d/sg-feats-%(tract)d.parq
-
+transformObjectCatalog_config:
+    persistable: Config
+    python: lsst.qa.explorer.postprocess.TransformObjectCatalogConfig
+    storage: ConfigStorage
+    template: config/TransformObjectCatalog
+consolidateObjectTable_config:
```

Comment:
Description for this and the one above would be welcome.  I imagine I'll learn what they are looking at obs_subaru, but for now I'm curious.